src/components/Environment/EnvironmentCard.tsx

```diff
@@ -94,15 +103,32 @@ const EnvironmentCard: React.FC<EnvironmentCardProps> = ({ environment, readOnly
               </Label>
             </DescriptionListDescription>
           </DescriptionListGroup>
+
           <DescriptionListGroup>
             <DescriptionListTerm>Deployment strategy</DescriptionListTerm>
             <DescriptionListDescription>
-              <Label>{getEnvironmentDeploymentStrategyLabel(environment)}</Label>
+              {getEnvironmentDeploymentStrategyLabel(environment)}
             </DescriptionListDescription>
           </DescriptionListGroup>
           <ApplicationEnvironmentStatus
             environment={environment as EnvironmentKindWithHealthStatus}
           />
+
+          <DescriptionListGroup>
```

Comment:
This can go before the application status. Having application status info at the end keeps it consistent for both views.